### PR TITLE
CC/Admin Outfits Update

### PR DESCRIPTION
**Pull request by Kyep**
This PR:
1. Replaces syndicate items in CC and other NT outfits with non-syndie alternatives, so that people don't wonder why centcom officers are walking around wielding syndicate gear (e.g: specops officer having dual-esword by default)
2. Fixes a bunch of bugs, non-OOP definitions, and general shittery in various outfits, such as the fact that CC officers' kit boxes won't accept items you take out of them being put back in, and many outfits lacking key items, such as singuloth knight not having a backpack, other outfits not having radios, outfits causing secHUDs to read you as "?" when they shouldn't, and so on.
3. Adds some new outfits, such as 'musician', 'unathi breacher' and 'ancient vampire'.

**Full changelist:**
Changes for both NT Navy Officer and NT Special Operations Officer
- Fixed issue where items removed from an officer kit box could not be re-inserted back into the officer kit box, because it couldn't fit all the items it spawned with.
- CentCom officer kit boxes also no longer contain advanced pinpointers (syndie contraband). Instead they contain a regular pinpointer (to find the captain/disk) and a centcom crew pinpointer (to find other people, regardless of suit sensors).

Changes for just NT Special Operations Officer:
- Replaced syndi double-esword and M-1911P with NT navy specops combat augmentation, a new arm aug that can deploy a M-1911P, hardlight blade (equivalent to esword but with different name/sprite), healing nanites, door remote, stun arm or flash. Effectively this puts all their weapons into their implant, which is immune to EMP.
- Fixed bug with 'officer jacket' that causes it to make vulp/taj/etc tails invisible when worn.
- Removed nodrop implant, as their good gear is already in their arm implant now, which means their gear is always nodrop.

Changes for just NT Navy Officer:
- Added a nutriment plus implant.

Changes for just NT Undercover Officer:
- Fixed the fact it was showing up as "?" on sec huds until ID was removed and reworn.
- Replaced SMG/silencer/ammo combination with NT navy specops combat augmentation, because its more flexible, less likely to result in dropped items, and allows concealed weapons without the use of syndi storage implant.
- Modified headset so that it grants access to ERT and specops channels, in case co-ordination is needed.
- Removed their custom black ID skin decal, as that made them too easy to identify.
- Removed their nodrop implant, for the same reason as SOO.

Other outfit changes:
- Syndicate Infiltrator: now uses actual H.equip_syndicate_infiltrator(), which ensures it has the exact same items as the normal command-spawned version does. This fixes several bugs with the outfit, including the fact it did not have proper radio, job listing, etc. More generally it prevents the two versions getting out of sync.
- Spy: replaced with new 'Syndicate Spy' outfit, which is the same as regular agent, but stealthier, has 40 TC instead of 20, is defined in a more OOP way, and is listed alongside the other syndi outfits rather than at the bottom by itself.
- NT VIP Guest, NT Navy Captain: now show correct NT icon on sec huds, ensured by calling H.sec_hud_set_ID() in post_equip().
- Space Pirate (all versions): now get toolbelts, combat gloves, radios, and flashlights, because without these items its impossible for them to radio each other, and they end up getting stuck in maintenance.
- Tunnel Clown: now gets a traitor clown grenade, because they were so weak before. Also a stunprod/cell, and a rubber ducky. Suggested by various admins.
- Mime Assassin: cane gun kit and accessories replaced by silenced pistol and accessories, because the cane gun was too unwieldy, and dragonsbreath shotguns aren't really an assassin weapon. Also fixed the issue where they would not show up correctly on sec huds.
- Musician: new outfit, essentially a singer getup with a variety of musical instruments in their bag. Civilian access.
- Soviet Admiral: given ID (standard type), because they were spawning without any ID at all.
- Assassin: deleted, on the grounds that it wasn't OOP (did not use syndicate/ template), was dead easy to spot (custom 'reaper' ID), and was fiddly to use (their gear spawned in a briefcase that required code fiddling which players don't know how to do).
- Singuloth Knight: given backpack (because they lacked one) containing survival box and flashlight.
- Dark Lord: given combat gloves and noslip black shoes.
- Ancient Vampire: new outfit, which is similar to dark lord, except that instead of a dsword they're instead a vampire with 999 blood, and some of their items are cosmetically different. 

:cl: Kyep
tweak: Tweaked many of the special admin outfits. This means centcom officers are less likely to be seen wielding illegal syndicate gear, and subtle flaws with many other outfits (such as pirates lacking radio, singulo knights lacking a backpack, several outfits not showing up on secHUDs correctly, etc etc etc) have been addressed.
/:cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
Alucard.

**Comment by TDSSS** [comment]
No more super-sweet silenced SMG? That was such a sexy weapon, not gonna lie.

**Comment by Kyep** [comment]
Yes, it was, but it had many issues:
- Reliant on ammo, thus could run out, required reloading, required storage for ammo, etc.
- Did not fit the pulse gun theme of CC, and since it was a ballistic weapon, sounded like a syndie gun.
- Lacked various key options, such as a stun setting.

File: code/game/gamemodes/vampire/vampire_powers.dm [hunk 1]
```diff
@@ -535,3 +535,75 @@
...
+	var/datum/objective/protect/protect_objective = new
+	protect_objective.owner = mind
+	protect_objective.target = M.mind
+	protect_objective.explanation_text = "Protect [M.real_name]."
+	mind.objectives += protect_objective
+	add_attack_logs(M, src, "Vampire-sired")
+	mind.make_Vampire()
+	revive()
+	Weaken(20)
+	return 1
```
Comment (variableundefined) [inline code/game/gamemodes/vampire/vampire_powers.dm]: Trailing return? Or just return TRUE

**Pull request merged**